ngraph/test/onnx/onnx_editor.cpp

```diff
@@ -676,6 +675,367 @@ NGRAPH_TEST(onnx_editor, subgraph__inputs_getter)
     EXPECT_EQ(editor.model_inputs(), (std::vector<std::string>{"conv1/7x7_s2_2:conv1/7x7_s2_1"}));
 }
 
+// HIGHT LEVEL API TESTS
+// INPUT EDGES TEST
+NGRAPH_TEST(onnx_editor, editor_api_select_input_edge_by_output_name_and_input_name)
+{
+    ONNXModelEditor editor{file_util::path_join(
+        SERIALIZED_ZOO, "onnx/model_editor/subgraph__inception_head.prototxt")};
+    const auto edge_mapper = editor.create_edge_mapper();
+
+    const InputEdge edge = edge_mapper.to_input_edge(EditorNode{EditorOutput{"conv1/7x7_s2_2"}},
```

Comment:
I would recommend changing the name `to_input_edge` to `find_input_edge`. Similar for `to_output_egde`

```suggestion
    const InputEdge edge = edge_mapper.find_input_edge(EditorNode{EditorOutput{"conv1/7x7_s2_2"}},
```

Reply:
done